src/model/Club.ts

```diff
@@ -5,6 +5,7 @@ export type ClubType = {
   clubImage: string;
   createdAt: string;
   ownerId: string;
+  memberCount: number;
```

Comment:
memberCount is NOT part of every club. It is just in the one endpoint if I am not mistaken.

God help you if you have added the counter to the database

Reply:
I have checked it. Your ass was on the line. Consider yourself lucky :D. Good job! :D